platform/platform_unix_impl.cpp

```diff
@@ -155,6 +155,16 @@ bool Platform::IsFileExistsByFullPath(string const & filePath)
 //static
 void Platform::DisableBackupForFile(string const & filePath) {}
 
+// static
+string Platform::GetWorkingDirectory() noexcept
+{
+  char path[PATH_MAX];
+  char const * const answer = getcwd(path, PATH_MAX);
```

Comment:
s/answer/dir/

Reply:
Done. In two places. For linux and for windows.